monkey/monkey_island/cc/server_utils/encryptor.py

```diff
@@ -51,7 +51,17 @@ def dec(self, enc_message):
         enc_message = base64.b64decode(enc_message)
         cipher_iv = enc_message[0 : AES.block_size]
         cipher = AES.new(self._cipher_key, AES.MODE_CBC, cipher_iv)
-        return self._unpad(cipher.decrypt(enc_message[AES.block_size :]).decode())
+        try:
```

Comment:
Whatever logic we add here will need unit tests.

Reply:
You might want to rebase on `develop` to pick up 3da4aee3d403f229b79c50e9cd54b789010ef707.